www_src/pages/element/element.jsx

```diff
@@ -109,7 +109,7 @@ render(React.createClass({
 
     return (<div>
       <Editor {...props} />
-      <button hidden={window.Android} onClick={()=>this.save()}>SAVE</button>
+      <button hidden={window.Android} onClick={()=>this.save()}>DEBUG:SAVE</button>
```

Comment:
at the moment the save function will try to call `e` param as a function, although we could add a typeof check